### ISSUE TITLE
disposeForm not called

### ISSUE THREAD

**Issue by manafire** [FilledStacks/boxtout#19]
In the [video on integrating sign in](https://youtu.be/Y-JawJ4m6Fg?t=1327), shouldn’t the disposeForm() method be called?   I’m debugging breakpoints and not seeing that generated method called.  Maybe it happens automatically?  I just noticed it because the TextEditControllers seem to “randomly” drop off and not pick up new values (in my integration of it) after awhile, switching between screens.  But that may have something to do with the emulator and hot reload, as restarting fixes it - for a little while.  

In any case, wondering if missing the dispose would cause a leak?

```
      onModelReady: (model) => listenToFormUpdated(model),
      onDispose: () => disposeForm(),  // <--- MISSING??
```

**Comment by manafire** [comment]
I'm also wondering if the disposeForm method needs to include calls to `removeListener`?

**Comment by FilledStacks** [comment]
@manafire yes. That is missing. It should be called there. I was probably going to add it in when I realised it's missing, if I ever did.
